### PR TITLE
Adds ability to skip checks

### PR DESCRIPTION
**Pull request by benjwadams**
Allows skipping of check methods (methods beginning with `check_`) via a
CLI flag of `-s` or `--skip-checks`. Addresses #202.

### CODE REVIEW

File: compliance_checker/runner.py [hunk 1]
```diff
@@ -16,7 +16,8 @@ class ComplianceChecker(object):
     the command line or can be used via import.
     """
     @classmethod
-    def run_checker(cls, ds_loc, checker_names, verbose, criteria, output_filename='stdout', output_format='stdout'):
+    def run_checker(cls, ds_loc, checker_names, verbose, criteria,
+                    skip_checks=[], output_filename='stdout', output_format='stdout'):
```
Comment (daf) [inline compliance_checker/runner.py]: You can't put `[]` in a method signature.

Comment (benjwadams) [inline compliance_checker/runner.py]: I'm aware of the issue when mutating an array, but this code shouldn't be modifying the default arg.  I could do a check for `None` instead if you'd like.

**Comment by benjwadams** [comment]
Oops, let me update the README with the new CLI flag.

File: compliance_checker/runner.py [hunk 2]
```diff
@@ -25,13 +27,16 @@ def run_checker(cls, ds_loc, checker_names, verbose, criteria, output_filename='
...
         @param  output_filename Path to the file for output
+        @param  skip_checks     Names of checks to skip
         @param  output_format   Format of the output
 
         @returns                If the tests failed (based on the criteria)
         """
         cs = CheckSuite()
         ds = cs.load_dataset(ds_loc)
-        score_groups = cs.run(ds, *checker_names)
+
```
Comment (lukecampbell) [inline compliance_checker/runner.py]: Not required but I prefer to do
```python
skip_checks = skip_checks or []
```

**Comment by lukecampbell** [comment]
Please change the CLI argument parser to use an array instead of `nargs=*`.  I would prefer users to have to write

```
--skip-checks <name> --skip-checks <name> --skip-checks <name>
```

rather than have the ambiguity of argument order because of the nargs.

File: README.md [hunk 3]
```diff
@@ -52,6 +52,11 @@ optional arguments:
                         Output format
   -o OUTPUT, --output OUTPUT
                         Output filename
+  -s {check1} [check2...] --skip-checks {check1} [check2...]
+                        Skips the any check functions contained within the
+                        check suites selected which have the same function
+                        name(s) as the string(s) specified.  May be specified
+                        multiple times in order to skip multiple checks.
```
Comment (lukecampbell) [inline README.md]: Could you update the README?

Comment (benjwadams) [inline README.md]: It was updated; specifically I added "May be specified multiple times in order to skip multiple checks."

Comment (benjwadams) [inline README.md]: Oh, let me fix the invocation for the command.

**Comment by lukecampbell** [comment]
@daf you all set?

**Comment by lukecampbell** [comment]
@daf could you take a gander over this way, I'd rather not merge when you have requested changes.

**Comment by kknee** [comment]
@daf can you take another look at this pull request?

**Pull request merged**